components/gc-subway/_screen-sm-max.scss

```diff
@@ -2,15 +2,24 @@
  * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
  * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
  *
- * Small view and under (screen only) - Variable $screen-sm-max <= 991 px
- *
- *	Provisional GC Subway map menu
+ * Small view and under (screen only) - Variable $screen-sm-max <= 991px
 */
 
-.provisional {
-	&.gc-subway {
-		dt, li {
-			max-width: initial;
+.gc-subway.provisional {
+	&:not(.gc-subway-index) {
+		h1 {
+			background-color: #fff;
+			border-bottom: none;
+			color: #555;
+			float: left;
+			font-size: 1.3em;
+			margin-left: -.5em;
+			margin-top: -.75em;
+			padding: 0px 20px 10px 0px;
```

Comment:
please add margin-right: 0.5em to avoid title cutting off entire top  bar if title word wraps
![image](https://user-images.githubusercontent.com/9367162/218214927-d06a3aa2-f62e-4869-82b9-d2668cf7de6d.png)